integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/management/CarbonLoadDataCommand.scala

```diff
@@ -251,7 +255,9 @@ case class CarbonLoadDataCommand(
         case ex: Exception =>
           LOGGER.error(ex)
           // update the load entry in table status file for changing the status to marked for delete
-          CarbonLoaderUtil.updateTableStatusForFailure(carbonLoadModel)
+          if (isUpdateTableStatusRequired && !table.isHivePartitionTable) {
```

Comment:
!table.isHivePartitionTable...this condition is not required while calling updateTableStatusForFailure as boolean flag isUpdateTableStatusRequired  will be set ti true only if the table is not a hive partition table